src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/PackageReferenceProject.cs

```diff
@@ -323,7 +344,6 @@ internal async Task<ProjectPackages> GetInstalledAndTransitivePackagesAsync(bool
             {
                 return targets
                     .SelectMany(target => target.Libraries)
-                    .Where(library => library.Type == LibraryType.Package)
```

Comment:
What types could be present here? Should this constrain it to Package and Project instead of removing the constraint?

Reply:
Looking more closely, this review I did was on a previous commit. Oops! :-) We talked offline and I left a few more comments. We should still consider adding a new data type to handle project references in the future and it would be good to keep track of where we create projects as package references to make it easier to modify if needed.